contracts/Commitment/PublicCommitment.sol

```diff
@@ -43,11 +44,14 @@ contract PublicCommitment is CommitmentBase {
     }
 
     function PublicCommitment(
+        IAccessPolicy _policy,
```

Comment:
You may move it to `BaseCommitment` class in that case, it seems that `AccessControlled` and `Reclaimable` are obligatory for any commitment contract.

Reply:
Done.